integration-libs/cdc/core/store/effects/cdc-user-addresses.effect.spec.ts

```diff
@@ -318,7 +320,7 @@ describe('CDC User Addresses effect', () => {
           });
```

Comment:
(I'm putting PR comment in this code line, because I cannot select the code above)
I believe you already have it on your checklist yet to do. Just in case, I'm leaving here a reminder here to improve the tests, according to the PR comment from previous PR https://github.com/SAP/spartacus/pull/16817#discussion_r1136317735

Reply:
Thanks @Platonn for the reminder, will add the code changes from https://github.com/SAP/spartacus/pull/16845/